apps/explorer/src/pages/transaction-result/transaction-summary/ObjectChanges.tsx

```diff
@@ -95,6 +88,41 @@ function Item({
     );
 }
 
+interface NFTItemProps {
+    description: string;
+    imageUrl: string;
+    objectId: string;
+}
+
+function NFTItem({ description, imageUrl, objectId }: NFTItemProps) {
```

Comment:
can we call this Object instead of `NFT`? Lots of things can have `display`.

Reply:
And this applies to the whole PR:
- NFT metadata is really `display`.
- NFT is really a displayable object.